Tests/SwiftDocCTests/Model/SemaToRenderNodeTests.swift

```diff
@@ -3452,4 +3452,23 @@ Document
             ])
         }
     }
+    
+    func testThematicBreak() throws {
+        let source = """
+
+        ---
```

Comment:
I know Swift-Markdown handles the parsing of thematic breaks, but would it be worth it to add a couple of tests that use `===` and `___` the other two styles you mentioned in this PR?